src/FileFormats/LP/LP.jl

```diff
@@ -285,9 +285,429 @@ end
 #   Base.read!
 #
 # ==============================================================================
+const COMMENT_REG = r"(.*?)\\(.*)"
```

Comment:
```suggestion
const _COMMENT_REG = r"(.*?)\\(.*)"
```

I'm going to be a bit pedantic on some naming (but I'll only comment on a couple). I know that the old code didn't have it. 

* https://jump.dev/JuMP.jl/dev/developers/style/#Naming
* https://jump.dev/JuMP.jl/dev/developers/style/#Exported-and-non-exported-names